build.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.7 -u
+#!/usr/bin/env python3
```

Comment:
We were getting away with using Python 2 because of the old Ubuntu base image. Luckily it seems we weren't using anything specific to v2